shared/modules/telemetry_module.go

```diff
@@ -0,0 +1,54 @@
+package modules
+
+import "github.com/prometheus/client_golang/prometheus"
+
+type (

```

Comment:
Affirmative.

---

I am in favor of maintaining consistency. I do this out of personal style and habit from older codebases.

Since we've now uncovered this, we can explicitly say that we use this style from now onwards.:

- `type` keyword for instance, no factoring.